src/MeshlessFV/MfvMusclSimulation.cpp

```diff
@@ -72,14 +72,26 @@ void MfvMusclSimulation<ndim>::MainLoop(void)
   else {
     this->ComputeBlockTimesteps();
   }
+  //WHY IS THIS HERE?!!!!
```

Comment:
This is redundant. If we were going to update levelneib when doing the godunov fluxes then we'd need it, but we don't.